semantics/language/common/typing/compatibility.k

```diff
@@ -352,8 +352,10 @@ module C-TYPING-COMPATIBILITY
           => t(Mods, functionType(T, P'))
      rule compositeFunctionType(T:Type, Mods:Set, .K, .List, P:List, _)
```

Comment:
Not at this point, but it may be worth beautifying a bit the C semantics at some later point.  Maybe have somebody else (an intern?) do it.  The C semantics is one of the oldest semantics we have, so no wonder that it may not take advantage of the latest developments.  So ok to merge.